src/actions/githubActions.js

```diff
@@ -1,16 +1,21 @@
-import timesComponentIssues from  './../data/mockgithubissuesTimesComponent.json';
-import dextroseIssues from  './../data/mockgithubissuesDextrose.json';
+import repositories from '../config/repositories';
+import requestBuilder from '../helpers/requestBuilder';
 
 export const GITHUB_FETCH_ISSUES = 'GITHUB_ISSUES_FETCH';
```

Comment:
Naming inconsistency can lead to confusion, these should be the same.

Reply:
que?